test/unit/alphabet/composite/semialphabet_any_test.cpp

```diff
@@ -9,6 +9,8 @@
 
 #include "../alphabet_test_template.hpp"
 #include "../alphabet_constexpr_test_template.hpp"
+#include "../semi_alphabet_test_template.hpp"
+#include "../semi_alphabet_constexpr_test_template.hpp"
```

Comment:
You added these test templates but nothing else in the code? Wrong includes, separated includes? what happened?

Reply:
This is funny because the other test templates were also not used. I did this via a big regex so I did not catch this